client/src/components/Country/LineGraph/LineGraph.tsx

```diff
@@ -4,12 +4,24 @@ import { ResponsiveLine } from "@nivo/line";
 export type Props = {
   results: any[];
   color?: boolean;
+  title: string;
 };
 
-export const LineGraph: React.FC<Props> = ({ results, color }) => {
+const scrollWidth = {
+  width: 10000
+}
+
+export const LineGraph: React.FC<Props> = ({ results, color, title }) => {
+  switch (title){
+    case 'Query':
+      scrollWidth.width = 20000;
+      break;
+    default:
+      scrollWidth.width = 1850;
+  }
   return (
-    <div style={{ height: 420, maxWidth: "100%" }}>
-      <ResponsiveLine
+    <div style={{ height: 420, maxWidth: "100%", overflow: 'scroll'}}>
```

Comment:
Could try applying overflow-x:'scroll' to avoid having an unnecessary y scroll bar. The y scroll bar is a bit distracting/confusing.

Reply:
Will fix that up right now.

Reply:
Actually, overflow-x isn't registered as a valid operation on my end. Very strange.